IdentityServer/Quickstart/Account/AccountController.cs

```diff
@@ -283,6 +284,60 @@ public async Task<IActionResult> Logout(LogoutInputModel model)
             return View("LoggedOut", vm);
         }
 
+        /// <summary>
+        /// Delete this user.
+        /// </summary>
+        /// <param name="returnUrl">The return URL to get back to the frontend.</param>
+        [HttpGet]
+        public async Task<IActionResult> Delete(string returnUrl)
```

Comment:
Async method is not doing anything I think? might remove it.

Reply:
I will remove the async, the redirect is not necessary anymore since the identity user and application user will be deleted at the same time now.